packages/compiler-vapor/src/generate.ts

```diff
@@ -403,16 +403,20 @@ function genOperation(oper: OperationNode, context: CodegenContext) {
 }
 
 function genSetProp(oper: SetPropIRNode, context: CodegenContext) {
-  const { pushFnCall, newline, vaporHelper, helper } = context
+  const { pushFnCall, pushMulti, newline, vaporHelper, helper } = context
 
   newline()
   pushFnCall(
-    vaporHelper('setAttr'),
+    vaporHelper('setDynamicProp'),
```

Comment:
If `oper.key` is static, the compiler should prefer to use the specific function directly, e.g `setClass`, `setStyle`, `setAttr`, instead of runtime checking.